pontos/github/api.py

```diff
@@ -385,3 +385,41 @@ def pull_request_files(
             ]
 
         return file_dict
+
+    def get_labels(
+        self,
+        repo: str,
+        issue: int,
+    ) -> List[str]:
+        """
+        Get all labels that are set in the issue/pr
+
+        Args:
+            repo:   GitHub repository (owner/name) to use
+            issue:  Issue/Pull request number
+
+        Returns:
+            List of existing labels
+        """
+        api = f"/repos/{repo}/issues/{issue}/labels"
+        response = self._request(api, request=requests.get)
+        labels = []
+
+        for f in response.json():
+            labels.append(f["name"])
+        return labels
```

Comment:
Maybe list comprehension?
```suggestion
        return [f["name"] for f in response.json()]
```